angular/projects/spark-angular/src/lib/components/sprk-tooltip/sprk-tooltip.component.spec.ts

```diff
@@ -273,4 +305,56 @@ describe('SprkTooltipComponent', () => {
       tooltipElement.classList.contains('sprk-c-Tooltip--bottom-right'),
```

Comment:
missing an expect

Reply:
The expect is on the previous line. Prettier reformatted a lot of these lines.